engine/src/main/scala/cromwell/engine/db/DataAccess.scala

```diff
@@ -148,7 +148,7 @@ trait DataAccess extends AutoCloseable {
   def queryWorkflowOutputs(id: WorkflowId)
                           (implicit ec: ExecutionContext): Future[Seq[MetadataEvent]] = {
     val uuid = id.id.toString
-    queryMetadataEventsWithWildcardKeys(uuid, s"${WorkflowMetadataKeys.Outputs}:".wrapNel, requireEmptyJobKey = true).
+    queryMetadataEventsWithWildcardKeys(uuid, s"${WorkflowMetadataKeys.Outputs}:%".wrapNel, requireEmptyJobKey = true).
```

Comment:
Is it definitely running?

Or maybe it genuinely wasn't working before, and this has fixed it?